### ISSUE TITLE
Localization

### ISSUE THREAD

**Issue by comradekingu** [kitsunyan/foxy-droid#17]
Would be nice to see this on Hosted Weblate (where G-Droid and F-Droid is to be found already).

**Comment by kitsunyan** [comment]
Yes, it's a good idea, though I never used it before.

**Comment by comradekingu** [comment]
@kitsunyan Send a request at https://hosted.weblate.org/hosting/ :)

**Comment by peat80** [comment]
So is foxy translatable soon? Would love to translate this app into german. 😎

**Comment by comradekingu** [comment]
@peat80 Meanwhile you can add a German translation by making a "values-de" folder with a "strings.xml" in it by coping the English source strings in [values/strings.xml](https://github.com/kitsunyan/foxy-droid/blob/master/src/main/res/values/strings.xml) over, like I did in https://github.com/kitsunyan/foxy-droid/pull/38 :)

**Comment by peat80** [comment]
@comradekingu thx, i tried with (my first pr) #40. 😀

**Comment by ghost** [comment]
@kitsunyan Is there any progress with translations? There are some pull requests with them.

**Comment by kitsunyan** [comment]
No, but I actually answered in this PR https://github.com/kitsunyan/foxy-droid/pull/38#issuecomment-752762809

I'll deal with them once I get a time.
